.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": ".NET Core Launch (console)",
+            "type": "coreclr",
+            "request": "launch",
+            "preLaunchTask": "build",
+            "program": "${workspaceFolder}/bin/Debug/<insert-target-framework-here>/<insert-project-name-here>.dll",
```

Comment:
You should insert values here

Reply:
![image](https://user-images.githubusercontent.com/1880826/64124388-97b98700-cdaf-11e9-9aa2-646adf8be170.png)

Reply:
As I see, there is another default launch.json task for asp net core application in this article: https://github.com/OmniSharp/omnisharp-vscode/issues/2673#issue-380290276